app/e2e/tests/user/user-settings.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from '@local/common/utils/fixtures';
+
+test.describe('User Settings Page', () => {
+    test('I can update my email', async({ userSettingsPage }) => {
+        // Go to User Settings
+        await userSettingsPage.goto();
+
+        // Enter new email address
+        await userSettingsPage.updateEmail(`${userSettingsPage.device}@example.com`);
+
+        // Verify email address was updated successfully
+        await expect(userSettingsPage.page.locator(`text=Current email: ${userSettingsPage.device}@example.com`)).toHaveCount(1);
```

Comment:
A bit nitpicky, but this locator might be long enough to warrant a method, would also make it a bit easier to read.